readme-vars.yml

```diff
@@ -6,6 +6,7 @@ project_url: "https://mullvad.net/en/browser"
 project_logo: "https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/mullvad-browser-logo.png"
 project_blurb: "The [Mullvad Browser]({{ project_url }}) is a privacy-focused web browser developed in a collaboration between Mullvad VPN and the Tor Project. It’s designed to minimize tracking and fingerprinting. You could say it’s a Tor Browser to use without the Tor Network. Instead, you can use it with a trustworthy VPN."
 project_lsio_github_repo_url: "https://github.com/linuxserver/docker-{{ project_name }}"
+project_categories: "Web Browser"
```

Comment:
Should probably add VPN as it has builtin wireguard support

Reply:
Done